source/agora/node/Validator.d

```diff
@@ -134,8 +134,9 @@ public class Validator : FullNode, API
             assert(0);
         }
 
+        const rand_seed = hashFull(1);
```

Comment:
Small note: The first commit does not use the preimages as the seed yet, because that change itself requires additional code. So I split it up into two commits.